pkg/kudoctl/kudoinit/types.go

```diff
@@ -0,0 +1,76 @@
+package kudoinit
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/kube"
+
+	"github.com/kudobuilder/kudo/pkg/version"
+)
+
+const (
+	DefaultNamespace      = "kudo-system"
+	defaultGracePeriod    = 10
+	defaultServiceAccount = "kudo-manager"
+)
+
+type InitStep interface {
+	Install(client *kube.Client) error
+	AsYamlManifests() ([]string, error)
+	AsArray() []runtime.Object
+}
+
+// Options is the configurable options to init
```

Comment:
I'd prefer `types.go` being about types only (no methods). Can we put `Options` and all methods in `options.go`? We can leave it in `/kudoinit/options.go` for now.

Reply:
Yeah, I like that. Done